tests/latency_test/udp_lat.c

```diff
@@ -2001,13 +2015,14 @@ int main(int argc, char *argv[])
 			{.name = "srv_num",		.has_arg = 1,   .val = OPT_CLIENT_WORK_WITH_SRV_NUM },
 			{.name = "force_unicast_reply",	.has_arg = 0,   .val = OPT_FORCE_UC_REPLY},
 			{.name = "mc-ttl",              .has_arg = 1,   .val = OPT_TTL},
+			{.name = "hw_timestamp",         .has_arg = 0,   .val = 'H'},
```

Comment:
1. I would have thought 'T' would be more appropriate for _t_ ime-stamping
2. Indentation